scripts/consensus_merge.py

```diff
@@ -527,6 +502,43 @@ def build_output_record(single_caller_variants, output_vcf, sample_names, hotspo
 
     return output_record
 
+def allowed_contigs_from_bed(bed_path, known_contigs):
+    """ Gets list of contigs from BED file
+        File must be tab-delimited with contig names in first field
+        Order of contigs will be same as in list of known contigs
+
+        Args:
+            bed_path (str): path to BED file
+            known_contigs (list): valid contig names (strings), presumably from VCF header
+
+        Raises:
+            IOError if bed_path does not point to a file
+                    if no contig names are identified
+                    if any contig name is not in known_contigs
+
+        Return:
+            list: list of contig names (strings)
+    """
+
+    if not os.path.isfile(bed_path):
+        raise IOError('Contig BED file not found at %s' % bed_path)
+
+    contig_names = []
+    with open(bed_path, 'r') as bed:
+        for line in bed:
+            fields = line.rstrip().split('\t')
+            name = fields[0]
+            if name in contig_names:
+                continue
+            if name not in known_contigs:
+                raise IOError('Contig %s not known from e.g. VCF header' % name)
+            contig_names.append(name)
+
+    if not contig_names:
+        raise IOError('No contig names identified in %s' % bed_path)
```

Comment:
Ok so we're just killing the job if there's a mislabeled contig. Not very permissive but at least the user has a descriptive error for this corner case.